packages/components/src/ResourcePicker/Resource/Resource.snap.test.js

```diff
@@ -47,6 +47,29 @@ describe('Resource component snaps', () => {
 			expect(wrapper.getElement()).toMatchSnapshot();
 		});
 
+		it('should render a regular Resource without author information', () => {
+			const collection = [
+				{
+					id: 0,
+					name: 'Title with few actions',
+					icon: 'talend-file-xls-o',
+				},
+			];
+			const props = {
+				parent: {
+					props: {
+						collection,
+						rowGetter: index => collection[index],
+					},
+				},
+				index: 0,
+			};
+
+			const wrapper = shallow(<Resource.WrappedComponent {...props} />);
+
+			expect(wrapper.getElement()).toMatchSnapshot();
```

Comment:
could you please avoid snapshots here. Just try to find the element and check it exists is enough.
no body check snapshots when they changes